axonivy/project-build-plugin
# Feature/xivy 622 integration test lifecycle

Providing @IvyWebTest POM infrastructure has become simple:

1. set <packaging>iar-integration-test</packaging>
2. define <build><testSourceDirectory>src_test</testSourceDirectory> ...
3. add some @IvyWebTest annotated files .. Filename is crucial should end with 'IT'. .... e.g. PortalLoginIT
